code/game/objects/effects/decals/Cleanable/tracks.dm

```diff
@@ -154,20 +154,49 @@ var/global/list/image/fluidtrack_cache=list()
 /obj/effect/decal/cleanable/blood/tracks/footprints
 	name = "wet footprints"
 	dryname = "dried footprints"
-	desc = "Whoops..."
-	drydesc = "Whoops..."
+	desc = "They look like still wet tracks left by footwear."
+	drydesc = "They look like dried tracks left by footwear."
 	coming_state = "human1"
 	going_state  = "human2"
 	amount = 0
 
+/obj/effect/decal/cleanable/blood/tracks/snake
+	name = "wet tracks"
+	dryname = "dried tracks"
+	desc = "They look like still wet tracks left by a giant snake."
+	drydesc = "They look like dried tracks left by a giant snake."
+	coming_state = "snake1"
+	going_state  = "snake2"
+	random_icon_states = null
+	amount = 0
+
+/obj/effect/decal/cleanable/blood/tracks/paw
+	name = "wet tracks"
+	dryname = "dried tracks"
+	desc = "They look like still wet tracks left by a mammal."
+	drydesc = "They look like dried tracks left by a mammal."
+	coming_state = "paw1"
+	going_state  = "paw2"
+	random_icon_states = null
+	amount = 0
+
+/obj/effect/decal/cleanable/blood/tracks/claw
+	name = "wet tracks"
+	dryname = "dried tracks"
+	desc = "They look like still wet tracks left by a reptile."
```

Comment:
Should probably generalize it to a clawed creature. Non-reptiles can easily have similiar feet e.g. avians.